File: Iniciante/1035 - Teste de Seleção 1.py
"""
Autor: Pierre Vieira
Data da submissão: 04/05/2018 16:40:21
"""
linha = input().split(" ")
a, b, c, d = linha
A = int(a)
B = int(b)
C = int(c)
D = int(d)
if B > C and D > A and C+D > A+B and C > 0 and D > 0 and A%2 == 0:
    print("Valores aceitos")
else:
    print("Valores nao aceitos")

File: C#/Iniciante/1035 - Teste de Seleção 1.cs
/*
* Autor: Pierre Vieira
* Data de submissão:
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int a, b, c, d;
            string[] linha;
            linha = Console.ReadLine().Split(' ');
            a = int.Parse(linha[0]);
            b = int.Parse(linha[1]);
            c = int.Parse(linha[2]);
            d = int.Parse(linha[3]);
            if (b > c && d > a && c + d > a + b && c > 0 && d > 0 && a % 2 == 0)
            {
                Console.WriteLine("Valores aceitos");
            }
            else
            {
                Console.WriteLine("Valores nao aceitos");
            }
        }
    }
}
